common/datavalues2/src/types/serializations/boolean.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_exception::ErrorCode;
+use common_exception::Result;
+
+use crate::prelude::*;
+
+pub struct BooleanSerializer {}
+
+impl TypeSerializer for BooleanSerializer {
+    fn serialize_value(&self, value: &DataValue) -> Result<String> {
+        if let DataValue::Boolean(x) = value {
+            if *x {
+                Ok("1".to_owned())
```

Comment:
How about decleare a const for `1` and `0` here?

Reply:
Ok, can be addressed.